SpatialGDK/Source/SpatialGDK/Public/Utils/SnapshotVersion.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+// SPATIAL_SNAPSHOT_VERSION is the current version of supported snapshots.
+//
+// If changes are made to the schema or content of snapshots then:
+//	(a) (optional) if the schema has changed, update SPATIAL_SNAPSHOT_SCHEMA_HASH,
+//			see the test
+//'GIVEN_snapshot_affecting_schema_files_WHEN_hash_of_file_contents_is_generated_THEN_hash_matches_expected_snapshot_version_hash'
+// which will provide the hash on failure
+//  (b) (mandatory) increment SPATIAL_SNAPSHOT_VERSION_INC
+
+constexpr uint32 SPATIAL_SNAPSHOT_SCHEMA_HASH = 958184518;
+constexpr uint32 SPATIAL_SNAPSHOT_VERSION_INC = 1;
+
+constexpr uint64 SPATIAL_SNAPSHOT_VERSION = ((((uint64)SPATIAL_SNAPSHOT_SCHEMA_HASH) << 32) | SPATIAL_SNAPSHOT_VERSION_INC);
```

Comment:
I think we would want this in SpatialServicesConstants.h instead?
The engine and gdk versions are in their own header files as a special case, to hopefully catch compilation errors earlier, but most of our other GDK-wide constants live in the above file.

Reply:
Or SpatialConstants